networkx/algorithms/dag.py

```diff
@@ -376,5 +387,70 @@ def antichains(G):
         while stack:
             x = stack.pop()
             new_antichain = antichain + [x]
-            new_stack = [t for t in stack if not ((t in TC[x]) or (x in TC[t]))]
+            new_stack = [
+                t for t in stack if not ((t in TC[x]) or (x in TC[t]))]
             antichains_stacks.append((new_antichain, new_stack))
+
+
+@not_implemented_for('undirected')
+def dag_longest_path(G):
+    """Returns the longest path in a DAG
+
+    Parameters
+    ----------
+    G : NetworkX DiGraph
+      Graph
+
+    Returns
+    -------
+    list : The longest path
```

Comment:
The Returns section for both functions needs one last fix.  The description of what is returned belongs on a second line with indentation, just as the Parameters section. Otherwise, HTML formatting will be incorrect.

More info: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt